main.py

```
@@ -116,9 +118,10 @@ def __init__(self):
         content_layout.addWidget(continue_reading)
         content_layout.addWidget(need_review)
         content_layout.addWidget(rss)
-        body_layout.addWidget(content_area)
 
-        self.setCentralWidget(body_widget)
+        self.splitter.addWidget(content_area)
+        self.splitter.setStretchFactor(1,1)
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)